src/containers/Dashboard/selectors.js

```diff
@@ -0,0 +1,39 @@
+import { createSelector } from 'reselect';
+
+const selectRandomBanner = state => state.get('randomBanner');
```

Comment:
This reducer is in Dashboard, so it should be called dashboardReducer, and in the above - state.get('dashboard');

banner is just a small part of that dashboard reducer.